### ISSUE TITLE
Changing the value returned by `padded_output_width()` in nerf_network.h breaks training

### ISSUE THREAD

**Issue by Soudini** [NVlabs/instant-ngp#659]
Steps to reproduce : 
* add any multiple of 16 to the output of NerfNetwork::padded_output_width() for instance: `return std::max(m_rgb_network->padded_output_width(), (uint32_t)4) + 16;`
* build and run

What happens :
* the initialization works fine/
* the second step is seemingly also fine.
* from the third training step onward, the loss become `nan`.

What should happen : 
From what I understand as padding is only added to to match the size of the tensor cores, changing its size should not impact in anyway the behaviour of the program. Am I wrong ?

**Comment by Soudini** [comment]
https://github.com/NVlabs/instant-ngp/blob/07d8e2ca7232f97397ab73af9b56c7db639d3445/include/neural-graphics-primitives/nerf_network.h#L447
